### PR TITLE
Password warning

### PR DESCRIPTION
**Pull request by Kodylow**
image

Changes password helper text to warning that the password is unrecoverable. Ran into this a couple times already with non-technicals setting up federations.

### CODE REVIEW

File: apps/guardian-ui/src/languages/en.json [hunk 1]
```diff
@@ -88,9 +88,9 @@
     "guardian-name": "Guardian name",
     "guardian-name-help": "This name will be shown to other Guardians",
     "admin-password": "Admin password",
-    "admin-password-help": "You'll need this every time you visit this page.",
+    "admin-password-help": "Use a strong password & back it up! You cannot recover this password!",
     "confirm-password": "Confirm password",
-    "confirm-password-help": "You'll need this every time you visit this page.",
+    "confirm-password-help": "Use a strong password & back it up! You cannot recover this password!",
```
Comment (wbobeirne) [inline apps/guardian-ui/src/languages/en.json:93]: Just searched for it, this isn't used anywhere, could be removed.

**Review by wbobeirne — APPROVED** [review]

File: apps/guardian-ui/src/components/SetConfiguration.tsx [hunk 2]
```diff
@@ -239,7 +239,11 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
             onChange={(ev) => setPassword(ev.currentTarget.value)}
             isDisabled={!!statePassword}
           />
-          <FormHelperText>{t('set-config.admin-password-help')}</FormHelperText>
+          <FormHelperText>
+            <Text color={theme.colors.yellow[400]}>
```
Comment (wbobeirne) [inline apps/guardian-ui/src/components/SetConfiguration.tsx:243]: Contrast ratio on this is pretty bad, might be worth bumping to 500 for text on white.

CleanShot 2024-01-09 at 16 17 33@2x

Comment (Kodylow) [reply apps/guardian-ui/src/components/SetConfiguration.tsx]: switched to 500 in next commit

File: apps/guardian-ui/src/languages/en.json (hunk 1 above)
Comment (Kodylow) [reply apps/guardian-ui/src/languages/en.json]: removed in next commit

**Review by wbobeirne — APPROVED** [review]

**Pull request merged**